build-tools/packages/build-cli/src/lib/package.ts

```diff
@@ -90,8 +136,31 @@ export async function npmCheckUpdates(
             throw new TypeError(`Expected an object: ${typeof result}`);
         }
 
-        for (const [pkgJsonPath, upgradedDeps] of Object.entries(result)) {
-            const jsonPath = path.join(repoPath, pkgJsonPath);
+        // npm-check-updates returns different data depending on how many packages were updated. This code detects the
+        // two main cases: a single package or multiple packages.
+        if (glob.endsWith("**")) {
```

Comment:
Can't a glob ending with a single `*` also match multiple packages?

Reply:
Hmmm, good point. fluid-build only supports the `**` glob today, because that was all that was needed at the time. I'll change this.